<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <!-- <meta http-equiv="Content-Type" content="text/html; charset=utf-8"> -->
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <!-- <meta property="og:image" content="https://mttarvina.github.io/TarvsHobbytronics/static/img/project_cover3.PNG"/> -->
    <meta property="og:description" content="Bacolod City National High School Online Reasearch Journal"/>

    <link rel="stylesheet" type="text/css" href="../../../static/css/bootstrap.min.css">
    <link rel="stylesheet" type="text/css" href="../../../static/css/base.css">
    <link rel="stylesheet" type="text/css" href="../../../static/css/prism.css"/> 
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
    <link rel="shortcut icon" type="image/jpg" href="../../../static/img/logo/amaranthine-logo.png"/>
    <title>Amaranthine: Volume 1 - B1 - Smut Disease Susceptibility of Hybrid Offsprings</title>
</head>


<body>
    <!-- NAVIGATION BAR -->
    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
        <div class="container-lg">
            <a class="navbar-brand" href="../../../index.html">
                <img class="d-inline-block align-center" src="../../../static/img/logo/amaranthine-logo.png" alt="logo" width="50" height="50">
            </a>
            <a class="navbar-brand" href="../../../index.html">Amaranthine</a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarCollapse">
                <ul class="navbar-nav me-auto mb-2 mb-md-0">
                    <li class="nav-item active">
                    <a class="nav-link" aria-current="page" href="../../../index.html">Home</a>
                    </li>
                    <li class="nav-item">
                    <a class="nav-link" href="../../home.html">Publications</a>
                    </li>
                    <li class="nav-item">
                    <a class="nav-link" href="../../../contact.html">Contact</a>
                    </li>
                    <li class="nav-item">
                    <a class="nav-link" href="../../../about.html">About</a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>


<!-- MAIN CONTENT HERE -->
<main class="container-fluid row mb-3">

<!-- Table of Contents -->
<div class="col-md-2">
<div class="leftpanel">
<a href="../home.html"><button type="button", class="btn btn-outline-primary">Volume 1 Articles</button></a>
<hr/>
<div class="tableofcontents btn-group dropend">
<button type="button" class="btn btn-outline-primary dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false">Table of Contents</button>

<!-- IMPORTANT: Table of contents will be auto generated here -->
<ul class="dropdown-menu">
<li><a class="dropdown-item" href="#Abstract">Abstract</a></li>
<li><a class="dropdown-item" href="#Introduction">Introduction</a></li>
<li><a class="dropdown-item" href="#Methodology">Methodology</a></li>
<li><a class="dropdown-item" href="#Preparation">+ Preparation of Crossbreed Accessions</a></li>
<li><a class="dropdown-item" href="#Extraction">+ Extraction of DNA Samples</a></li>
<li><a class="dropdown-item" href="#Evaluation">+ Evaluation of the Concentration and Purity of the Samples</a></li>
<li><a class="dropdown-item" href="#Preparation">+ Preparation of Working Solutions</a></li>
<li><a class="dropdown-item" href="#Polymerase">+ Polymerase Chain Reaction</a></li>
<li><a class="dropdown-item" href="#Polyacrylamide">+ Polyacrylamide Gel Electrophoresis and Gel Documentation</a></li>
<li><a class="dropdown-item" href="#Molecular">+ Molecular Marker Scoring and Cluster Analysis</a></li>
<li><a class="dropdown-item" href="#Polyacrylamide">+ Polyacrylamide Gel Electrophoresis</a></li>
<li><a class="dropdown-item" href="#Cluster">+ Cluster Analysis Using R Software</a></li>
<li><a class="dropdown-item" href="#Conclusion">Conclusion</a></li>
<li><a class="dropdown-item" href="#Recommendations">Recommendations</a></li>
<li><a class="dropdown-item" href="#References">References</a></li>
</ul>

</div>
</div>
</div>

<!-- IMPORTANT: Main content will be auto generated here-->
<div class="main-content col-md-10">
<h1 id='smut-disease-susceptibility-of-hybrid-offsprings-from-resistant-and-susceptible-saccharum-spp-sugarcane-parental-accessions-using-smut-resistant-primer-srp-6'>Smut Disease Susceptibility of Hybrid Offsprings from Resistant and Susceptible Saccharum spp (Sugarcane) Parental Accessions using Smut Resistant Primer (SRP) 6</h1>
<br>

<figure><table class="table table-striped">
<thead>
<tr><th>Authors</th><th>Julianne Marc M. Tamayo, Tisha Gabrielle P. Hormigos, Lyanne Clich S. Supe, John Moises G. Relles, Dynah Fatima I. Discaya, Meliza Z. Mana-ay, Phillip Raymund R. De Oca, Joey Ian C. Singson</th></tr></thead>
<tbody><tr><td>Volume</td><td>1</td></tr><tr><td>Date Published</td><td>February 23, 2023</td></tr><tr><td>Date Updated</td><td>April 10, 2023</td></tr></tbody>
</table></figure>
<hr />
<br><br>
<h2 id="Abstract">Abstract</h2><p class="text-justify">Smut disease susceptibility of different crossbred offsprings from resistant and susceptible sugarcane accessions were determined using Smut Resistant Primer 6 (SRP6). DNA samples were extracted from each hybrid accession. Their concentration, purity, and qualities were determined. Screening used PCR, then Polyacrylamide Gel Electrophoresis with the results subjected to Molecular Marker Scoring and Cluster Analysis using R Software. The results revealed that 42 offsprings were proven susceptible to Smut. When accession Phil 99-1793, resistant, is used as a male parent, some of the hybrids become susceptible, no matter which accession it is paired with. When compared to another primer with the same offsprings, accession 219 showed the most susceptible samples, while 217 showed none. The library generated by the marker scoring and R analysis from the study can be optimized to facilitate breeding in the country.</p>
<hr />

<br><br>
<h2 id="Introduction">Introduction</h2><p class="text-justify">Sugarcane farms mostly encounter diseases caused by either biotic or abiotic factors [1][2]. One of the most severe is Smut, a sugarcane fungal disease brought by the fungus Sporisorium scitamineum [3][4][5][6] . The disease can cause heavy losses to cane production, sugar content, and juice quality [7][8][9] . Canes that are infected indicate substantial metabolic changes leading to the formation of a whip-like sorus made up of a combination of fungal hyphae and plant tissues [10] . The said disease hinders the target output of the Sugarcane Regulatory Administration anchored on the Sugarcane Industry Roadmap 2020, including: an increase in the sugarcane area, farm productivity, and sugar yield [11]. To safeguard stalk cuttings from the disease, triazole fungicides have been widely employed, in the likes of propiconazole and triadimefon [3][6][12] . The treatment, however, may be difficult to dispose of in large volumes and has logistics limitations as well as side effects on the human body [13][14] . Simple sequence repeats (SSRs), also called microsatellites, are proven and useful markers frequently utilized as a resourceful tool in plant breeding programs and evolutionary studies due to their significant ability in indicating cultivar diversity [15][16][17][18] . A microsatellite is a span of repetitive genetic sequence in which the repeating unit contains from one to six bases [19] . Microsatellites are used to locate a gene or a mutation responsible for a given trait or disease. They can be amplified for identification by PCR and can be used as molecular markers [20] . Through microsatellites, the sugarcane varieties being released are screened and tested through their genes or DNA. The Smut Resistant Primer used for significant association with susceptibility in this study is SRP6 [2] . This primer is a serine/threonine kinase protein. This protein plays a paramount role in cellular homeostasis [21] . Short fragments of DNA with a specific sequence corresponding to 1 the target DNA to be identified and amplified are the primers [22] . In this study, susceptibility to the Smut disease of different accessions of sugarcane that are offsprings of recessive and susceptible crossbred accessions was determined. This study seeks to help in the development of markers to quicken the release of varieties where the selection usually spans a period of 12 years [23] , thus increasing the yield and quality of sugarcane in the industry.</p>

<br><br>
<h2 id="Methodology">Methodology</h2><p class="text-justify">Hybrid sugarcane offsprings crossbred from the following accessions were evaluated for susceptibility to the Smut disease with SRP6: Hybrid 216 is the offspring of Phil 93-118-1207 (female, susceptible) and Phil 99-1793 (male, resistant), 217 of Phil 93-118-1207 (female, susceptible) and Phil 93-2349 (male, resistant), 218 of Phil 99-1793 (female, resistant) and Phil 93-118-1207 (male,susceptible), 219 of Phil 2004-1011 (female, susceptible) and Phil 99-1793 (male, resistant), 220 of Phil 2005-1097 (female, susceptible) and Phil 99-1793 (male, resistant), and 221 of Q102 (female, susceptible) and Phil 99-1793 (male, resistant). This was conducted in the Sugar Genome Laboratory La Granja Agricultural Research and Extension Center.</p>
<br><br>
<h3 id="Preparation">Preparation of Crossbreed Accessions</h3><p class="text-justify">The parental accessions were chosen wherein one is resistant and the other is susceptible. Cross-pollination was employed where the male is positioned above the female and placed in a Hawaiian solution: a mixture of acids formulated to give life to sugarcane every time it provides pollen. For the pollen of the male sugarcane to fall directly to the female every time it pollinates, the female sugarcane is placed in the soil below the male sugarcane. The aim is to identify the gene sequence responsible for the sugarcane variety’s susceptibility to Smut. The seedlings were grown in a greenhouse nursery.</p>
<br><br>
<h3 id="Extraction">Extraction of DNA Samples</h3><p class="text-justify">After six months, ten (10) samples each from six crosses were obtained. The samples were freeze-dried with ALPHA 1-2 LDplus Freeze Dryer. The process is done for extraction efficiency and total removal of moisture from the leaves. Extraction of the freeze-dried samples used the CTAB method, eliminating contaminants, polysaccharides, and polyphenols by applying the cationic detergent CTAB and Polyvinylpyrrolidone: the polyphenol binding agent.</p>
<br><br>
<h3 id="Evaluation">Evaluation of the Concentration and Purity of the Samples</h3><p class="text-justify">NanoDrop 8000 Ultraviolet-Visible Spectrophotometer was used to check the concentration and purity of the extracted samples. The quality of the extracted samples was checked through Agarose Gel Electrophoresis. It followed three protocols: first, a gel was prepared with an agarose concentration adjusted to match the size of the DNA fragments to be separated. Sample wells were loaded with the DNA samples. The gel was run at 100V for a period of time until optimal separation was attained. Finally, using a Bio-Rad Gel Doc XR+, the gel was visualized directly after being illuminated with UV light [23].</p>
<br><br>
<h3 id="Preparation">Preparation of Working Solutions</h3><p class="text-justify">The formula C1V1=C2V2 was used to prepare volume 1 of the working solution, wherein C stands for concentration and V for volume. Volume 1 is the volume of the sample to be taken from the stock solution. The calculated amount of Volume 1 was added individually in separate tubes and 2 added sNP (sterile nanopure) water to each sample.</p>
<br><br>
<h3 id="Polymerase">Polymerase Chain Reaction</h3><p class="text-justify">PCR mix: SnP water, ICR buffer, dNTPs, MgCl2, Primer forward, Primer reverse, and Taq Polymerase were prepared. The reagents were pipetted in a 1 mL tube in order previously stated. The PCR mix prepared for the initial working solutions were added. The stages of the PCR Screening were Denaturation (95°C), Annealing (51°C), Elongation (72°C), and 12°C. The cycle was repeated 30 times for DNA to be visualized.</p>
<br><br>
<h3 id="Polyacrylamide">Polyacrylamide Gel Electrophoresis and Gel Documentation</h3><p class="text-justify">Polyacrylamide gel electrophoresis (PAGE) was used. In this, for binding to proteins and giving them a negative charge, Sodium dodecyl sulfate (SDS) was utilized. Then, the proteins were electrophoretically sorted by size through a gel matrix composed of polyacrylamide in an electric field. Bio-Rad GelDoc XR Plus was then used for the Gel Documentation Process.</p>
<br><br>
<h3 id="Molecular">Molecular Marker Scoring and Cluster Analysis</h3><p class="text-justify">Each of the PAGE run was subjected to molecular marker scoring. Only evident and apparent bands were scored following the procedures of Relles et al. Bands were scored one (1) for the presence and zero (0) for the absence of a DNA fragment. The presence or absence of the amplified band in all genotypes indicates similarity, whereas the presence on one and absence on another indicates dissimilarity[2] . The software R studio was used where the binary data were encoded and the dendrogram was generated. Results and Discussions Purity of the Extracted Samples Figure 1. Initial 60 extracted samples’ purity Figure 1 shows the results for the purity of the 60 extracted samples. The purity level values range from 1.8 to 2.0. When the purity level hits lower than the said range value, it means that the DNA extracted contains protein contamination. If the purity level is above 2.0, the DNA extracted contains phenol contamination. When the purity level reaches 2.5-3.0, then the polysaccharide was extracted instead of DNA in the hybrid leaf sample. The sample with the highest purity value based upon the figure is 220-4. The sample with the lowest is 218-1, with a value of 0</p>
<p class="text-justify">[INSERT FIGURE 1 HERE]</p>
<br><br>
<h3 id="Polyacrylamide">Polyacrylamide Gel Electrophoresis</h3><p class="text-justify">[INSERT FIGURE 2 HERE]</p>
<p class="text-justify">[INSERT FIGURE 3 HERE]</p>
<p class="text-justify">Based on the results from the Polyacrylamide Gel Electrophoresis, the bands are scored according to their positions in the ladder. The bands that can be visualized are those apparent only. They were counted and scored to know which variety has the highest amount of bands. The hybrid offsprings were labeled one (1) with the presence of bands and zero (0) when the bands are non-existent. Based on the results, 15 hybrid offsprings have bands, and the other 42 have zero (0) bands. Hybrid offsprings 218-2 and 218-6 have the highest number of alleles, which is 5. Hybrid offspring 218-5 and 221-5 have 4 alleles; hybrid offspring 218-7, 220-4, and 220-5 have 3 alleles; hybrid offspring 217-9, 219-7, 221-1, and 221-4 have alleles; and hybrid offspring 217-8, 221-7, 221-9, and 221-10 have 1 allele; and the other 42 hybrid offsprings have none. The average number of alleles per hybrid offspring is 0.64 alleles. SRP6-1 has 5 alleles, SRP6-2 has the most alleles detected with 14 alleles, SRP6-3 has 3, SRP6-4 has 8, SRP6-7 has 3, SRP6-8 has 1, SRP6-9 has 3, and SRP6-10 has 3. The primers with the lowest number of alleles present are SRP6-5 and SRP6-6 with 0 alleles. The maximum size of the alleles detected is 180bp, and the minimum is 130bp.</p>
<br><br>
<h3 id="Cluster">Cluster Analysis Using R Software</h3><p class="text-justify">The dendrogram is a result of using the R Statistical Program. It is a visual representation that provides an overview of the genetic distances among accessions. Statistical analysis was done with the aid of the R Statistical Language and Environment. Encoding of binary data utilized this software in which it was interpreted to create the dendrogram. Cluster 1 shows the 15 offsprings that do not have a susceptible trait. Cluster 2 shows the 42 crossbred offsprings that have the probability of being susceptible. These offsprings have genetic markers in their genes that make them susceptible to the disease Smut. The Dissimilarity in the dendrogram signifies how genetically distant they are based on the alleles detected. The values below the dendrogram show the percentage of their dissimilarity. The closer the branch is to 1.0 (100%), the more genetically distant it is from the other accessions.</p>
<p class="text-justify">[INSERT FIGURE 4 HERE]</p>
<p class="text-justify">The offsprings with the lowest dissimilarity is 218-4 and 218-5, garnering 0.21 (21%). This means that their genetic distance is close to each other. The offsprings with the highest dissimilarity of 1.0 (100%) are 216-, 216-2, 216-3, 216-4, 216-5, 216-6, 216-7, 216-8, 216-9, 216-10, 217-1, 217-2, 217-3, 217-4, 217-5, 217-6, 217-8, 218-2, 218-3, 218-7, 218-8, 218-9, 219-1, 219-2, 219-3, 219-4, 219-5, 219-6, 219-8, and 219-9, 219-10, 220-1, 220-2, 220-3, 220--6, 220-7, 220-8, 220-10, 221-1, 221-4, 221-6. They are 100% genetically distant from each other.</p>
<br><br>
<h2 id="Conclusion">Conclusion</h2><p class="text-justify">It was confirmed after tests that 42 crossbred offsprings had been proven susceptible to Sugarcane Smut disease with SRP6 as the primer used. Offspring 216, the cross between Phil 93-118-1207 (female, susceptible) and Phil 99-1793 (male, resistant) was shown to have the most susceptible samples. All ten samples of the offspring were found to be susceptible. 221, on the other hand, the cross between Q102 (female, susceptible) and Phil 99-1793 (male, resistant) was shown to have the least susceptible samples, with a total value of four.</p>
<p class="text-justify">However, for better evaluation and interpretation of results, the researchers compared two different primers, SRP3 and SRP6, with the same crossbred sugarcane accessions to determine the common accessions to both primers that are Susceptible to the disease Smut. The results show that there are 16 offsprings that are highly likely to be susceptible to Sugarcane Smut: 216-1 (1), 216-4 (4), 216-6 (6), 216-8 (8), 218-3 (22), 218-4 (23), 218-10 (29), 219-3 (32), 219-4 (33), 219-5 (34), 219-6 (35), 219-9 (38), 220-1 (40), 220-2 (41), 221-4 (51), and 221-5 (52).</p>
<p class="text-justify">According to the gathered and compared results, only the hybrid offspring 217, the cross between Phil 93-118-1207 (female, susceptible) and Phil 93-2349 (male, resistant), showed no susceptible samples. The rest of the hybrids show susceptibility. The specific hybrid offspring with the most number of susceptible samples is 219, the cross between Phil 2004-1011 (female, susceptible) and Phil 99-1793 (male, resistant), followed by 216, the cross between Phil 93-118-1207 (female, susceptible) and Phil 99-1793 (male, resistant).</p>
<p class="text-justify">Moreover, when the accession Phil 99-1793 resistant is used as the male parent, some offspring show susceptibility no matter which female accession it is paired with. When Phil 93-118-1207, susceptible, is used as the female parent, the offsprings show no susceptibility, but when it is used as a male, some offsprings become susceptible.</p>
<p class="text-justify">This study may be beneficial to the industry as the data gathered can be optimized for the creation of markers. This will quicken the pace of releasing varieties, hence improving the yield and quality of sugarcane in the industry.</p>
<br><br>
<h2 id="Recommendations">Recommendations</h2><p class="text-justify">From the results of the study, researchers recommend growing seedlings into the field and checking their phenotypic expression. Hypothetically speaking, they should be susceptible based on the garnered data. However, to confirm it, the varieties must be tested and observed in the field.</p>
<br><br>
<h2 id="References">References</h2><ol start='' >
<li>Relles, J. M. G., Discaya, D. F. I., Mana-Ay, M. Z., Meneses, N. S., Entima, R. G., &amp; Armones, R. T. Association Analysis of Developed Simple Sequence Repeat (SSR) Markers to Fungal Disease Resistance.</li>
<li>Hasan, S. S., Solomon, S., Baitha, A., Singh, M. R., Sah, A. K., Kumar, R., &amp; Shukla, S. K. (2015). CaneDES: A web-based expert system for disorder diagnosis in sugarcane. Sugar tech, 17(4), 418-427</li>
<li>Comstock, J. C. 2000. Smut. Pages 181-185 in: A Guide to Sugarcane Diseases. P. Rott, R. A. Bailey, J. C. Comstock, B. J. Croft, and A. S. Saumtally, eds. CIRAD and ISSCT, Montpellier, France</li>
<li>Rott, P., Bailey, R. A., Comstock, J. C., Croft, B. J., &amp; Saumtally, A. S. (2000). A guide to sugarcane disease. CIRAD. ISSCT.</li>
<li>Croft, B. J., Magarey, R. C., Allsopp, P. G., Cox, M. C., Willcox, T. G., Milford, B. J., &amp; Wallis, E. S. (2008). Sugarcane smut in Queensland: arrival and emergency response. Australasian Plant Pathology, 37(1), 26-34.</li>
<li>Bhuiyan, S. A., Croft, B. J., James, R. S., &amp; Cox, M. C. (2012). Laboratory and field evaluation of fungicides for the management of sugarcane smut caused by Sporisorium scitamineum in seedcane. Australasian Plant Pathology, 41(6), 591-599.</li>
<li>Que, Y., Xu, L., Wu, Q., Liu, Y., Ling, H., Liu, Y., Zhang, Y., Guo, J., Su, Y., Chen, J., Wang, S., and Zhang, W. (2014). Genome sequencing of Sporisorium scitamineum provides insights into the pathogenic mechanisms of sugarcane smut. BMC genomics, 15(1), 996.</li>
<li>Sundar, A. R., Barnabas, E. L., Malathi, P., Viswanathan, R., Sundar, A. R., &amp; Barnabas, E. L. (2012). A mini-review on smut disease of sugarcane caused by Sporisorium scitamineum. Botany, 2014, 226.</li>
<li>Hasan, S. S., Solomon, S., Baitha, A., Singh, M. R., Sah, A. K., Kumar, R., &amp; Shukla, S. K. (2015). CaneDES: A web-based expert system for disorder diagnosis in sugarcane. Sugar tech, 17(4), 418-427.</li>
<li>Marques, Joao Paulo &amp; Appezzato-da-Glória, Beatriz &amp; Piepenbring, Meike &amp; Massola, Nelson &amp; Monteiro-Vitorello, Claudia &amp; Vieira, Maria-Lucia. (2016). Sugarcane smut: Shedding light on the development of the whip-shaped sorus. Annals of Botany. 119. mcw169. 10.1093/aob/mcw169.</li>
<li>Relles, J. M. G., Laurena, A. C., Diaz, M. G. Q., &amp; Lalusin, A. G. (2018). Development of Microsatellite Markers From Sugarcane (Saccharum Officinarum) L.) Phil 97-3933. International Journal of Agriculture, Environment, and Bioresearch.</li>
<li>Bailey, R. A. (1983). The effect of soil and seedcane applications of Triadimefon on the incidence of sugarcane smut (Ustilago scitaminea). In Proc S Afr Sug Technol Ass (Vol. 57, pp. 99-104).</li>
<li>Bhuiyan, S. A., Croft, B. J., &amp; Tucker, G. R. (2015). New method of controlling sugarcane smut using flutriafol fungicide. Plant disease, 99(10), 1367-1373.</li>
<li>Simmons, M. (2017, November 14). Triazole — toxicity, side effects, diseases and environmental impacts. Retrieved from <a href='https://pesticides.news/2017-11-14triazole-toxici' target='_blank' class='url'>https://pesticides.news/2017-11-14triazole-toxici</a> ty-side-effects-diseases-and-environmental-impa cts.html.</li>
<li>Sharon, D., Adato, A., Mhameed, S., Lavi, U., Hillel, J., Gomolka, M., ... &amp; Epplen, J. T. (1995). DNA fingerprints in plants using simple-sequence repeat and minisatellite probes. HortScience, 30(1), 109-112.</li>
<li>Hameed, U., Pan, Y. B., Muhammad, K., Afghan, S., &amp; Iqbal, J. (2012). Use of simple sequence repeat markers for DNA fingerprinting and diversity analysis of sugarcane (Saccharum spp) cultivars resistant and susceptible to red rot.</li>
<li>Levi, A., &amp; Rowland, L. J. (1997). Identifying blueberry cultivars and evaluating their genetic relationships using randomly amplified polymorphic DNA (RAPD) and simple sequence repeat-(SSR-) anchored primers. Journal of the American Society for Horticultural Science, 122(1), 74-78.</li>
<li>Varshney, R. K., Graner, A., &amp; Sorrells, M. E. (2005). Genic microsatellite markers in plants: features and applications. TRENDS in Biotechnology, 23(1), 48-55.</li>
<li>Marín-García, J. (2011). Post-genomic cardiology. Academic Press.</li>
<li>Kalia, R. K., Rai, M. K., Kalia, S., Singh, R., &amp; Dhawan, A. K. (2011). Microsatellite markers: an overview of the recent progress in plants. Euphytica, 177(3), 309-334.</li>
<li>Capra, M., Nuciforo, P. G., Confalonieri, S., Quarto, M., Bianchi, M., Nebuloni, M., ... &amp; Draetta, G. F. (2006). Frequent alterations in the expression of serine/threonine kinases in human cancers. Cancer research, 66(16), 8147-8154.</li>
<li>Garibyan, L., &amp; Avashia, N. (2013). Research techniques made simple: polymerase chain reaction (PCR). The Journal of investigative dermatology, 133(3), e6.</li>
<li>Bischoff, K. P., &amp; Gravois, K. A. (2004). The development of new sugarcane varieties at the LSU AgCenter. J. Am. Soc. Sugar Cane Technol, 24, 142-164.</li>

</ol>


</div>

</main>


<!-- FOOTER -->
<footer class="footer mt-auto py-3 bg-light">
    <div class="container">
        <span class="text-muted">Powered by: <a href="https://www.facebook.com/tarvshobbytronics" target="_blank">Tarvs' Hobbytronics</a></span>
    </div>
</footer>        

<script src="../../../static/js/bootstrap.bundle.min.js"></script>
<script src="../../../static/js/prism.js"></script>
</body>
</html>